docs/configuration.md

```diff
@@ -209,12 +204,12 @@ php artisan up
 |`MAIL_USERNAME`|Username of SMTP server|`null`|
 |`MAIL_PASSWORD`|Password of SMTP server|`null`|
 |`MAIL_FROM_ADDRESS`|"From" address|`hello@example.com`|
-|`MAIL_FROM_ADDRESS`|"From" name|`Example`|
-|`QUEUE_CONNECTION`|Connection for queues, `sync`, `database`, `beanstalkd`, `sqs` or `redis`|`sync`|
+|`MAIL_FROM_NAME`|"From" name|`Example`|
 |`SESSION_DRIVER`|Driver for sessions, `file`, `cookie`, `database`, `apc`, `memcached`, `redis`, `dynamodb` or `array`|`file`|
-|`SESSION_LIFETIME`|session lifetime in minutes|`120`|
-|`TRUSTED_PROXIES`|Trusted proxy ID addresses|`null`|
+|`SESSION_LIFETIME`|Idle session timeout in minutes; the session will need to be reinitialized once it expires|`120`|
```

Comment:
```suggestion
|`SESSION_LIFETIME`|Idle session expiration in minutes; the session will need to be reinitialized once it has expired|`120`|
```

Please don't use the word "timeout". I know that "timeout" and "expiration" can mostly be used as synonyms, but this one has already led me on a wrong path once. In a technical context, for me a request/response pair is something which can time out, but a session can expire.

Moreover, I am not sure if the "will" is 100% correct. According to my schoolbook English it is not, because this is nothing which will happen in an indefinite future, but more a mathematical condition which always holds so it should use present tense. But native speakers to the front (@kamil4 and @d7415).

Reply:
"will" is fine. "the session has to be reinitialised when it expires" would also work, but I wouldn't say it was better.